rest/v8/sticker.ts

```diff
@@ -0,0 +1,51 @@
+import type { APISticker, APIStickerPack } from '../../payloads/v8/index';
+
+export type RESTGetAPIStickerResult = APISticker;
+
+export interface RESTGetNitroStickerPacksResult {
+	sticker_packs: APIStickerPack[];
+}
+
+export type RESTGetAPIGuildStickersResult = APISticker[];
+
+export type RESTGetAPIGuildStickerResult = APISticker;
+
+export interface RESTPostAPIGuildStickerFormDataBody {
+	/**
+	 * Name of the sticker (2-30 characters)
+	 */
+	name: string;
+	/**
+	 * Description of the sticker (up to 100 characters)
+	 */
+	description?: string;
+	/**
+	 * The name of a unicode emoji representing the sticker's expression
+	 */
+	tags: string;
+	/**
+	 * The sticker file to upload
+	 */
+	file: unknown;
+}
+
+export type RESTPostAPIGuildStickerResult = APISticker;
+
+export interface RESTPatchAPIGuildStickerJSONBody {
+	/**
+	 * Name of the sticker (2-30 characters)
+	 */
+	name?: string;
+	/**
+	 * Description of the sticker (up to 100 characters)
+	 */
+	description?: string;
+	/**
+	 * The name of a unicode emoji representing the sticker's expression
+	 */
+	tags: string;
```

Comment:
Is this required?

Reply:
yeah https://github.com/discord/discord-api-docs/pull/3127#discussion_r652056949

Reply:
wait oops this is PATCH... shouldn't be